testing/trino-testing/src/main/java/io/trino/testing/containers/BaseTestContainer.java

```diff
@@ -125,6 +129,26 @@ public void start()
         startContainer();
     }
 
+    public void runCommandInContainer(String... commandAndArgs)
+    {
+        String commandString = Arrays.asList(commandAndArgs)
```

Comment:
How about this `Joiner.on(" ").join(commandAndArgs)`

Reply:
Ok seems shorter.